src/index.html

```diff
@@ -1,28 +1,38 @@
 <!DOCTYPE html>
 <html>
 <head>
+  <title><%= htmlWebpackPlugin.options.title %></title>
```

Comment:
Pretty sure this should be below the `meta` since some browsers only look at the first 1024 bytes before choosing an encoding.